nengo/conftest.py

```diff
@@ -18,8 +19,7 @@
 
 
 def pytest_configure(config):
-    rc.reload_rc([])
-    rc.set('decoder_cache', 'enabled', 'False')
+    matplotlib.use('agg')
```

Comment:
> but `tox` won't install the required modules

I consider it the user's job to set up their matplotlib properly.

> 'agg' should work on (almost?) all systems for the file output we're producing.

Sometimes, when I'm testing, I use `plt.show` to actually pop up a plot window. But I guess that's considered a hack, and not something we need to support. Also, I can still hack it by just importing matplotlib myself. I would like to be pretty sure that 'agg' will work on almost all systems for file output, but if you have good reason to believe that I'm fine with this. I don't know enough about it myself to say.